.github/workflows/beaker.yml

```diff
@@ -39,6 +39,11 @@ on:
         default: docker
         required: false
         type: string
+      domain:
+        description: The domain that will be used for the beaker instances
+        default: ${{ github.run_attempt }}-${{ github.run_id }}-${{ github.sha }}.example.com
```

Comment:
Be careful that a label can be 63 characters and FQDN at most can be 255 characters. I don't know the exact rendered format, but that can cause problems in places. If needed, you may be able to hash the 3 parameters together.

Reply:
I tested this directly in https://github.com/betadots/puppet-hdm/pull/18 and the generated fqdn is `debian10-64-puppet7.1-3677086063-29766356511a720115832ce34594425c381658a8.example.com`